src/components/LinkConnect.tsx

```diff
@@ -29,28 +32,71 @@ export const LinkConnect = (props: LinkConfiguration) => {
 
     if (props.settings) {
       sdkTypeScript += `
-        window.accessTokens='${JSON.stringify(props.settings.accessTokens || {})}';
-        window.transferDestinationTokens='${JSON.stringify(props.settings.transferDestinationTokens || {})}';
+        window.accessTokens='${JSON.stringify(
+          props.settings.accessTokens || {}
+        )}';
+        window.transferDestinationTokens='${JSON.stringify(
+          props.settings.transferDestinationTokens || {}
+        )}';
       `;
     }
 
-    return sdkTypeScript;
-  }, [
-    props.settings,
-  ]);
+    let styleScript = '';
 
-  const SDKWrapperComponent = props.renderViewContainer ? SDKViewContainer : SDKContainer;
+    if(darkTheme) {
+      styleScript = `
+        document.body.style.backgroundColor = ${DARK_THEME_COLOR};
+      `;
+    }
```

Comment:
we do not need this. commented lines 46-50 and it was working fine. so let's remove this part